geode-docs/tools_modules/gfsh/command-pages/query.html.md.erb

```diff
@@ -21,7 +21,11 @@ limitations under the License.
 
 Run queries against Geode regions.
 
-Run the specified OQL query as a single quoted string and displays results in pages allows to move between pages. If limit is not set in the query, then a default limit of 1000 (derived from GFSH environment variable APP\_FETCH\_SIZE) will be applied. Page size is derived from GFSH environment variable APP\_COLLECTION\_LIMIT (default value=20).
+If a limit restricting the result size is not set in the query,
+then a default limit of the gfsh environment variable `APP_FETCH_SIZE`
```

Comment:
The default size is given in the gfsh section titled 'Useful gfsh Shell Variables.'  I'd like to have that default value given in only 1 place in the documentation, such that if the value ever changes, we only need to update a single location.  If that is OK with you, I'll consider that you approve this PR.